docs/reference/cli-reference/cli-ref-sign.md

```diff
@@ -29,7 +29,12 @@ where `<package(s)>` is one or more `.nupkg` files.
 
 - **`-CertificateFingerprint`**
 
-  Specifies the SHA-1 fingerprint of the certificate used to search a local certificate store for the certificate.
+  Specifies the fingerprint of the certificate used to search a local certificate store for the certificate.
```

Comment:
I find this a little hard to read, can it just be "fingerprint" instead of "fingerprint of the cert" ?

```suggestion
  Specifies the fingerprint to be used to search for the certificate in a local certificate store.
```